errors/2.py

```diff
@@ -0,0 +1,83 @@
+class DivisionByZeroError(Exception):
+    def __str__(self):
+        print("Division by zero")
+
+
+class InvalidInputError(Exception):
+    def __str__(self):
+        print("Invalid input")
+
+
+def example1():
+    for i in range(3):
+        try:
+            x = int(input("Enter a number: "))
+            y = int(input("Enter another number: "))
+
+            if y == 0:
+                raise DivisionByZeroError("Error: Division by zero is not allowed.")
+
+            print(x, '/', y, '=', x / y)
+
+        except ValueError:
+            print("Error: Please enter integers.")
```

Comment:
w przypadku większych aplikacji unikamy używania print do wypisywania błędów, za to korzystamy z python logging
```py
import logging
logging.basicConfig(level=logging.ERROR)

logging.info("test info")
logging.error("test loggera")
logging.warning("test loggera")
```